pkg/trace/info/stats.go

```diff
@@ -155,15 +167,108 @@ func (ts *TagStats) publish() {
 	metrics.Count("datadog.trace_agent.receiver.events_extracted", eventsExtracted, tags, 1)
 	metrics.Count("datadog.trace_agent.receiver.events_sampled", eventsSampled, tags, 1)
 	metrics.Count("datadog.trace_agent.receiver.payload_accepted", requestsMade, tags, 1)
+
+	for reason, count := range ts.TracesDropped.tagValues() {
+		metrics.Count("datadog.trace_agent.normalizer.traces_dropped", count, append(tags, "reason:"+reason), 1)
+	}
+
+	for reason, count := range ts.TracesMalformed.tagValues() {
+		metrics.Count("datadog.trace_agent.normalizer.traces_malformed", count, append(tags, "reason:"+reason), 1)
+	}
+}
+
+// inlineNonZeroTagValues serializes the entries in this map into format "key1: value1, key2: value2, ...", sorted by
+// key to ensure consistent output order. Only non-zero values are included.
+func inlineNonZeroTagValues(statsMap map[string]int64) string {
```

Comment:
I've learned and felt that it's better to name functions based on what they do, not how or where they are used. This function takes a map and returns a string representation of it. Can we call it as such? Maybe `mapString` or `mapToString` ?
```go
// mapString returns a string representation of the given map, eluding empty values.
func mapToString(m map[string]int64) string {
```

Reply:
Sure, updated.